src/templates/sidebar.html

```diff
@@ -17,34 +17,62 @@ <h3 class="hidden-xs">
 
   {#    Group Menus #}
   <div class="nav list-group hidden-xs sidebar-menu" id="lg-menu-main">
-    <div class="clearfix">
+    <div class="clearfix dropdown">
      <a id="quests-menu" class="list-group-item left-side
         {% if '/quests/' in request.path_info and not '/approvals/' in request.path_info and not '/inprogress/' in request.path_info and not '/drafts/' in request.path_info%}active{% endif %}"
        href="{% url 'quests:quests' %}"><i class="fa fa-shield fa-fw"></i>&nbsp; Quests</a>
-    {% if request.user.is_staff %}
+
+      <div id="dropdown-button-wrapper"> <!-- isolates dropdown button highlight -->
+        <a id="quests-menu-dropdown" 
+          class="list-group-item right-side dropdown-toggle text-center" 
+          data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+          <i class="fa fa-ellipsis-v"></i></a>
+        <ul class="dropdown-menu dropdown-menu-right">
+          {% if request.user.is_staff %}
+            <li><a href="{% url 'quests:categories' %}"><i class="fa fa-compass"></i>&nbsp; Campaigns</a></li>
```

Comment:
Add `fa-fw` to the icons to get text to line up vertically after an icon (font awesome fixed-width, makes all icon elements the same width by adjusting padding margins)

Reply:
fa-fw tag added to fontawesome icons.